src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -477,7 +527,10 @@ export default () => {
       workspaces={workspacesSortedByTime}
       onCreate={onCreate}
       onDelete={onDelete}
+      setMessage={setMessage}
+      message={message}
```

Comment:
This message would be better rendered separate from the Workspaces component. Instead, pass down a ref that a component down the tree will render to using a portal

Reply:
not sure why this is showing up in this pr might have been from a messed up rebase, but this is already merged